ow_plexil/src/plans/FaultDependenciesModel.xml

```diff
@@ -0,0 +1,95 @@
+<System>
+  <Subsystems>
+
+    <Subsystem Name="Power">
+      <Faults>
+        <Fault Name="LOW_STATE_OF_CHARGE">
+          <Impacts Name="Power" Type="Subsystem"/>
+        </Fault>
+        <Fault Name="INSTANTANEOUS_CAPACITY_LOSS">
+        </Fault>
+        <Fault Name="THERMAL_FAULT">
+          <Impacts Name="Power" Type="Subsystem"/>
+        </Fault>
+      </Faults>
+
+      <Impacts Name="Arm" Type="Subsystem"/>  
+      <Impacts Name="Antenna" Type="Subsystem"/>  
+      <Impacts Name="Camera" Type="Subsystem"/>  
+    </Subsystem>
+
+    <Subsystem Name="Arm">
+      <Faults>
+        <Fault Name="HARDWARE">
+          <Impacts Name="Arm" Type="Subsystem"/>
+        </Fault>
+        <Fault Name="TRAJECTORY_GENERATION">
+          <Impacts Name="Arm" Type="Subsystem"/>
+        </Fault>
+        <Fault Name="COLLISION">
+          <Impacts Name="Arm" Type="Subsystem"/>
+        </Fault>
+        <Fault Name="E_STOP">
+          <Impacts Name="Arm" Type="Subsystem"/>
+        </Fault>
+        <Fault Name="NO_FORCE_DATA">
+          <Impacts Name="Arm" Type="Subsystem"/>
+        </Fault>
+        <Fault Name="FORCE_TORQUE_LIMIT">
+          <Impacts Name="Arm" Type="Subsystem"/>
```

Comment:
Suggest removing these since they are not supported in OceanWATERS.  Outside this PR, but we should insert comments to the same effect in the ROS action file.

Reply:
That sounds fine to me, @AstroStucky .